lib/Twig/Extension/Core.php

```diff
@@ -213,11 +213,11 @@ public function getTests()
             new Twig_SimpleTest('even', null, array('node_class' => 'Twig_Node_Expression_Test_Even')),
             new Twig_SimpleTest('odd', null, array('node_class' => 'Twig_Node_Expression_Test_Odd')),
             new Twig_SimpleTest('defined', null, array('node_class' => 'Twig_Node_Expression_Test_Defined')),
-            new Twig_SimpleTest('sameas', null, array('node_class' => 'Twig_Node_Expression_Test_Sameas', 'deprecated' => true, 'alternative' => 'same as')),
+            new Twig_SimpleTest('sameas', null, array('node_class' => 'Twig_Node_Expression_Test_Sameas', 'deprecated' => true, 'deprecatedSince' => '1.21', 'alternative' => 'same as')),
```

Comment:
Instead of adding another entry, what about using the existing `deprecated` option: `'deprecated' => '1.21'`? We can also keep `true` for when we don't know the deprecated version and to keep BC.